Quintor/src/main/java/com/example/quintor/NavbarController.java

```diff
@@ -14,7 +14,7 @@ public class NavbarController {
     private BorderPane borderPane;
 
     public void initialize() {
-//        loadPage("dashboard");
+        loadPage("fileOverview");
     }
 
     public void dashboardButton() {
```

Comment:
Deze methode moet ook nog werkend, zodat de knop in de navbar ook naar het dashboard/fileoverview kan navigeren